src/NuGet.Core/NuGet.ProjectManagement/Utility/BuildIntegratedProjectUtility.cs

```diff
@@ -41,7 +41,16 @@ public static class BuildIntegratedProjectUtility
         /// </summary>
         public static async Task<LockFile> GetLockFileOrNull(BuildIntegratedNuGetProject buildIntegratedProject)
         {
-            var lockFilePath = await buildIntegratedProject.GetAssetsFilePathAsync();
+            string lockFilePath;
+            try
+            {
+                lockFilePath = await buildIntegratedProject.GetAssetsFilePathAsync();
+            }
+            catch (ProjectNotLoadedException)
+            {
+                return null;
```

Comment:
can we just skip executing scripts in case there is a project which isn't being nominated? wouldn't that be a safe fail?